deployment/k8s/charts/aggregates/migrationAssistantWithArgo/values.yaml

```diff
@@ -47,12 +47,23 @@ images:
     tag: latest
     pullPolicy: IfNotPresent
 
-snapshotBucketConfiguration:
-  create: true
-  bucketName: snapshot
+# Default AWS values are dummy values for LocalStack usage
+aws:
+  eksEnabled: false
```

Comment:
This flag still makes me nervous.  Maybe it could be called something like 'configureAwsEksResources'?  It's probably fine, but I have to look at all of the usages every time to remember and to make sure that I'm OK with it.